community/bolt/src/test/java/org/neo4j/bolt/v1/runtime/internal/ErrorReporterTest.java

```diff
@@ -75,20 +88,23 @@ public void shouldNotReportOOMErrors()
         // Then
         assertThat( error.status(), equalTo( (Status) Status.General.OutOfMemoryError ) );
         assertThat( error.message(),
-                equalTo(  "There is not enough memory to perform the current task. Please try increasing " +
-                          "'dbms.memory.heap.max_size' in the process wrapper configuration (normally in 'conf/neo4j-wrapper" +
-                          ".conf' or, if you are using Neo4j Desktop, found through the user interface) or if you are running an embedded " +
-                          "installation increase the heap by using '-Xmx' command line flag, and then restart the database." ) );
-        provider.assertNoLoggingOccurred();
+                equalTo( "There is not enough memory to perform the current task. Please try increasing " +
+                        "'dbms.memory.heap.max_size' in the process wrapper configuration (normally in " +
+                        "'conf/neo4j-wrapper" +
+                        ".conf' or, if you are using Neo4j Desktop, found through the user interface) or if you are " +
+                        "running an embedded " +
+                        "installation increase the heap by using '-Xmx' command line flag, and then restart the " +
+                        "database." ) );
+        provider.assertContainsMessageContaining( cause.getClass().getName() );
     }
 
     @Test
-    public void shouldNotReportStackOverflowErrors()
+    public void shouldNotReportStackOverflowErrorsInUserLogButShouldInInternalLog()
```

Comment:
Poor name. The test body doesn't resemble the test name, the "why" will be logs. Suggest `shouldReportStackOverflowErrorsInInternalLog`